.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parser": "babel-eslint",
+  "extends": "standard",
+  "globals": {
+    "atom": false,
+    "afterEach": false,
+    "beforeEach": false,
+    "describe": false,
+    "fdescribe": false,
+    "xdescribe": false,
+    "expect": false,
+    "it": false,
+    "fit": false,
+    "xit": false,
```

Comment:
Linting should actually fail on `fdescribe`, `xdescribe`, `fit` and `xit`. If one of those functions are committed that means that the specs broken.